apps/web/components/eventtype/EventSetupTab.tsx

```diff
@@ -157,13 +165,36 @@ export const EventSetupTab = (
       return true;
     });
 
+    const defaultValue = isManagedEventType
+      ? locationOptions.find((op) => op.label === t("default"))?.options[0]
+      : undefined;
+
+    const locationAvailable =
```

Comment:
This is to be able to know if the selected location is actually available as an option for the user, in order to show go install the app if not.